requirements.txt

```diff
@@ -0,0 +1,8 @@
+# Dependencies maintained by OCCRP
```

Comment:
Sorry, but "one more thing": since we have this file now, can we take the chance to pin these to exact versions?